10376.xml

```diff
@@ -0,0 +1,202 @@
+				The record use CBOR format encoded as following:
+				1. array(Record Index 1, Record Timestamp 1, Record Value 1)
+				2. array(Record Index 2, Record Timestamp 2, Record Value 2)
+				...
+				n. array(Record Index n, Record Timestamp n, Record Value n)
+				Record Timestamp is a 32-bit integer representing the number of seconds since Jan 1st, 1970 in the UTC time zone. Record Value can be any type and described by CBOR type. Value can also be an array of subvalues.]]>
+				</Description>
+			</Item>
+			<Item ID="7">
+				<Name>Record Head Index</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the latest record. This index is incremented upon every new activity's record. Total recordings inside the device can be calculated by Record Head Index - Record Tail Index.]]>
+				</Description>
+			</Item>
+			<Item ID="8">
+				<Name>Record Tail Index</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the oldest record stored in device. This index is adjusted(incremented) by device based on it's memory space availability.]]>
+				</Description>
+			</Item>
+			<Item ID="9">
+				<Name>Record Read Index</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the index of the last read record. Device will increment this index upon successful record delivery to server(via manual read or scheduled dispatch). Server may modify this index to traverse within available records in device.]]>
+				</Description>
+			</Item>	
+			<Item ID="10">
+				<Name>Record Dispatch Start Mask</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the start time to deliver Record resource. If Record resource is currently being observed, this resource enables accumulation of the Record changes before being dispatch (rather than delivering Record immediatly as it changes). As per Start Mask, Dispatch Start Mask must adhere to following string format:
```

Comment:
What's the expected time zone for the time? Same applies to other time related fields. Did you consider using RFC3339?

Reply:
I assumed it would be enough to refer to mandatory Device [/3/0/14] object for UTC offset (perhaps I need to mention this in the description). 
Or do you think it's better to add another optional Read resource containing device UTC offset specially for this particular instance object?

Reply:
I think both are OK options as long as you make it explicit. Alternative would be to use RFC3339 format that supports this. Perhaps the device object UTC offset reference makes most sense.

Reply:
I have updated the description to include a reference to the Device object's UTC Offset resource. Thanks for your input.